socorro/scripts/fetch_crash_data.py

```diff
@@ -135,17 +137,53 @@ def fetch_crash(fetchdumps, outputdir, api_token, crash_id):
             with open(fn, 'wb') as fp:
                 fp.write(data)
 
+    if fetchprocessed:
+        # Fetch processed crash data
+        print('Fetching processed %s' % crash_id)
+        resp = session.get(
+            host + '/api/ProcessedCrash/',
+            params={
+                'crash_id': crash_id,
+                'format': 'meta',
+            },
+            headers=headers,
+        )
+
+        # Handle 404 and 403 so we can provide the user more context
+        if resp.status_code == 404:
+            raise CrashDoesNotExist(crash_id)
+        if api_token and resp.status_code == 403:
+            raise BadAPIToken(resp.json().get('error', 'No error provided'))
+
+        # Raise an error for any other non-200 response
+        resp.raise_for_status()
+
+        # Save processed crash to file system
+        processed_crash = resp.json()
+        fn = os.path.join(outputdir, 'v1', 'processed_crash', crash_id)
+        create_dir_if_needed(os.path.dirname(fn))
+        with open(fn, 'w') as fp:
+            json.dump(processed_crash, fp, cls=JsonDTEncoder, indent=2, sort_keys=True)
+
 
 def main(argv=None):
     parser = argparse.ArgumentParser(
         formatter_class=WrappedTextHelpFormatter,
         description=DESCRIPTION.strip(),
         epilog=EPILOG.strip(),
     )
+    parser.add_argument(
+        '--host', default='https://crash-stats.mozilla.org',
+        help='host to pull crash data from; this needs to match SOCORRO_API_TOKEN value'
+    )
```

Comment:
Adding `--host` brings this in line with `fetch_crashids` which already has this flag.

One thing we could do is move the `DEFAULT_HOST` up to `socorro/scripts/__init__.py` as a constant used across scripts that set the host. I don't know how much that helps, but I remember we discovered an unenthusing proliferation of host setting when we adjusted .com to .org.